README.md

```diff
@@ -7,3 +7,104 @@ This is a suitcase subpackage for reading a particular file format.
 ```
 pip install suitcase-jsonl
 ```
+
+## Common use cases
+The common use case for this file format in streaming mode is to attach a
+RunRouter to the RunEngine (RE) using the following lines:
+
+```python
+# create a factory that will use the serializer to create on-the-fly files.
+def factory(name, start_doc):
+    # the following creates a serializer that will close the file when the stop
+    # document is sent to it.
+    serializer = suitcase.jsonl.Serializer(directory='...',
+                                         file_prefix='...')
```

Comment:
Thanks for applying my suggestions, @awalter-bnl! I missed one more line after adding a couple of blank spaces:
```suggestion
                                           file_prefix='...')
```